rfs_test/TEST_accountservice_schema.py

```diff
@@ -18,283 +18,470 @@
+    rq_headers = self.request_headers()
+    rq_body = {
+        'UserName': 'test_id',
+        'Password': 'test_pswd',
+        'RoleId': 'Administrator'
+    }
+
+     json_payload, headers, status = self.http_GET(self.sut_toplevel_uris['AccountService']['url'], rq_headers, authorization)
+    testAccountId = json_payload['Id']
+
+def deleteDummyAccount(self):
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    json_payload, headers, status = self.http_DELETE(self.sut_toplevel_uris['Accounts']['url'] + testAccountId, rq_headers, authorization)
+
+
+
+def failAuthorization(self):
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    rq_body = {
+        'UserName': 'test_id',
+        'Password': 'wrong_pswd'
+    }
+
+    json_payload, headers, status = self.http_POST(self.sut_toplevel_uris['SessionService/Sessions']['url'], rq_headers, rq_body, authorization)
+
+    # The status code for authentication credentials included with this request being missing or invalid.
+    return status == 401
+    
+# Question: Does this method redquire authorization in order to perform the Reset action on LogService ?
+def clearLog(self): 
+    authorization = 'on'
+    rq_headers = self.request_headers()
+    rq_body = {
+      
+    }
+
+    json_payload, headers, status = self.http_POST(self.sut_toplevel_uris['Managers/MultiBladeBMC/LogServices/Log/Actions/LogService.Reset']['url'], rq_headers, rq_body, authorization)
+    
+    return status == 200 or status == 201 or status == 202 or status == 204
+
 
 ###################################################################################################
-# Name: Assertion_ACCO101(self, log) :Account Service                                            
-# Assertion text: 
-#The value of this property shall be a boolean indicating whether this service is enabled.  If 
-#this is set to false, the AccountService is disabled.  This means no users can be created, 
-#deleted or modified.  Any service attempting to access the Account Service, like the Session 
-#Service, will fail accessing.  Thus new sessions cannot be started with the service disabled 
-#(though established sessions may still continue operating).  Note: this does not affect Basic 
-#AUTH connections.
+# Name: Assertion_ACCO101(self, log) :Account Service
+# Assertion text:
+# The value of this property shall be a boolean indicating whether this service is enabled.  If
+# this is set to false, the AccountService is disabled.  This means no users can be created,
+# deleted or modified.  Any service attempting to access the Account Service, like the Session
+# Service, will fail accessing.  Thus new sessions cannot be started with the service disabled
+# (though established sessions may still continue operating).  Note: this does not affect Basic
+# AUTH connections.
 ###################################################################################################
 
 def Assertion_ACCO101(self, log):
```

Comment:
The steps for this should be the following:

1. Find the AccountService
2. Check if the AccoutService has a "ServiceEnabled" property
    If it does not, end the test
3. Perform a PATCH on the AccountService to set ServiceEnabled to False
    If this fails due to the property being Read Only, end the test
4. Perform a POST on the Accounts collection found in the AccountService
    If a response in the 4XX or 5XX range is returned, the test passes; otherwise the test fails
5. Perform a PATCH on the AccountService to set ServiceEnabled back to true